src/orquestra/sdk/secrets/_api.py

```diff
@@ -18,18 +19,22 @@ def _translate_to_zri(workspace_id: WorkspaceId, secret_name: str) -> str:
 def get(
     name: str,
     *,
-    workspace_id: t.Optional[str] = None,
-    config_name: t.Optional[str] = None,
+    workspace_id: t.Optional[WorkspaceId] = None,
+    config_name: t.Optional[ConfigName] = None,
 ) -> str:
     """Retrieves secret value from the remote vault.
 
+    For more information about the supported config/runtime names, visit the
+    `Runtime Configuration guide
+    <https://docs.orquestra.io/docs/core/sdk/guides/runtime-configuration.html>`_.
+
     Args:
         name: secret identifier.
         workspace_id: ID of the workspace. Using platform-defined default if omitted -
             - currently it is personal workspace
-        config_name: config entry to use to communicate with Orquestra Platform.
-            Required when used from a local machine. Ignored when
-            ORQUESTRA_PASSPORT_FILE env variable is set.
+        config_name: name of the runtime use to communicate with Orquestra Platform.
```

Comment:
Nitpick: grammar, I think this should be one of the following:
```suggestion
        config_name: name of the runtime to use to communicate with Orquestra Platform.
```
```suggestion
        config_name: name of the runtime used to communicate with Orquestra Platform.
```

Reply:
Thanks 🤦 I put the blame on dust in my keyboard switch